cubeb-ffi/src/ffi.rs

```diff
@@ -0,0 +1,674 @@
+use libc::{c_long,c_void};
+use std::ops;
+use std::default::Default;
+
+#[macro_export]
+macro_rules! log_internal {
+    ($level: expr, $msg: expr) => {
+        #[allow(unused_unsafe)]
```

Comment:
Yes. If you place log! or logv! in an unsafe block then you'll get a warning that the unsafe is not needed.